src/App.tsx

```diff
@@ -25,7 +25,6 @@ import {
   editorOptions,
   sampleValue,
   sampleZodSchema,
-  STORAGE_KEY,
```

Comment:
this was unused

Reply:
ok. we can remove it. Maybe we should consider adding a linter

Reply:
I thought there was already eslint. Are you keen trying biome?

Reply:
https://biomejs.dev/

Reply:
Yeah I am very curious. I heard it but never tried it before.

Reply:
let's do this in another pr

Reply:
https://github.com/marilari88/zod-playground/pull/57